mmv1/products/networksecurity/ClientTlsPolicy.yaml

```diff
@@ -0,0 +1,160 @@
+# Copyright 2023 Google Inc.
```

Comment:
This file should be put into the folder products/networkservices.

Reply:
Why is that? This is part of the networksecurity API.

Reply:
Oh. Yes, you are right. Sorry about that.